ignite/services/scaffolder/component.go

```diff
@@ -139,6 +139,11 @@ func checkComponentCreated(appPath, moduleName string, compName multiformatname.
 		typesToCheck["Msg"+compName.UpperCamel] = componentMessage
 		typesToCheck["MsgSend"+compName.UpperCamel] = componentPacket
 	}
+	// Make all names lower case to handle cases of the same name with different cases
+	for k, v := range typesToCheck {
+		delete(typesToCheck, k)
+		typesToCheck[strings.ToLower(k)] = v
+	}
```

Comment:
At this point, could we just build the `typesToCheck` map with only lowercase keys?

Reply:
yeah, I thought the idea of adding the lower call to each was more trouble than just restructuring the keys after the map was built. I'll switch it up if you prefer so.

Reply:
I think we should just build the strings without calling `strings.ToLower()`:

example:
```
typesToCheck["msgsend"+compName.LoweCase] = componentPacket
```

Reply:
yup, that's the best course